### PR TITLE
update prefix to sf3_

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: Domain/DoctrineAclCache.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
  */
 class DoctrineAclCache implements AclCacheInterface
 {
-    const PREFIX = 'sf2_acl_';
+    const PREFIX = 'sf3_acl_';
```
Comment (fabpot) [inline Domain/DoctrineAclCache.php]: what about just using `sf_acl_`?

Comment (HeahDude) [inline Domain/DoctrineAclCache.php]: @fabpot why not indeed, but this is still a bc break

Comment (HeahDude) [inline Domain/DoctrineAclCache.php]: so this need to be done for other PRs to avoid future bc break on 4.0

Comment (HeahDude) [inline Domain/DoctrineAclCache.php]: @fabpot commits ready, I can close these PRs and open the new ones with `sf_` prefix when you agree with this change

**Pull request closed without merging**